### PR TITLE
Disable broken macOS CI job, which is unused

### PR DESCRIPTION
**Pull request by anmenaga**
# PR Summary

Copying this change from release branch of v7.0.10

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [X] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [X] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [X] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [X] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

**Comment by daxian-dbw** [comment]
@anmenaga, @TravisEz13, the macOS daily build has been successful, for quite some time. Is the change in this PR really need to be ported to GitHub master? maybe it only affects the 7.0.10 release branch?

**Comment by TravisEz13** [comment]
I think we can delete this package now.  I don't think it is used.

**Comment by daxian-dbw** [comment]
Assign to @adityapatwardhan to determine if this change should be merged.

**Comment by adityapatwardhan** [comment]
No this is not used anymore. The test packages created here were used for code coverage runs we used to run. We create these packages in release build for release automation testing.

File: .vsts-ci/mac.yml [hunk 1]
```diff
@@ -91,16 +91,17 @@ stages:
...
-  - job: CodeCovTestPackage
-    displayName: CodeCoverage and Test Packages
-    pool:
-      vmImage: macOS-latest
-    steps:
-    - pwsh: |
-        Import-Module .\tools\ci.psm1
-        New-CodeCoverageAndTestPackage
-      displayName: CodeCoverage and Test Package
+# this is no longer used
```
Comment (adityapatwardhan) [inline .vsts-ci/mac.yml:94]: Can we just delete this? We have history if we need to bring this back.

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: .vsts-ci/mac.yml [hunk 2]
```diff
@@ -91,16 +91,17 @@ stages:
...
+#  jobs:
+#  - job: CodeCovTestPackage
+#    displayName: CodeCoverage and Test Packages
+#    pool:
+#      vmImage: macOS-latest
+#    steps:
+#    - pwsh: |
+#        Import-Module .\tools\ci.psm1
+#        New-CodeCoverageAndTestPackage
+#      displayName: CodeCoverage and Test Package
```
Comment (TravisEz13) [inline .vsts-ci/mac.yml:107]:
```suggestion
```

File: .vsts-ci/mac.yml (hunk 1 above)
Comment (TravisEz13) [reply .vsts-ci/mac.yml]: Done.

**Review by adityapatwardhan — APPROVED** [review]

**Pull request merged**